File: 0301_0400/0388/0388.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, m = (int(x) for x in f.readline().split())
    s = 0
    p = []
    for i in range(n):
        p.append([int(x) for x in f.readline().split()])
    row = [min(p[i][j] for j in range(m)) for i in range(n)]
    col = [max(p[i][j] for i in range(n)) for j in range(m)]
    for i in range(n):
        for j in range(m):
            if p[i][j] == row[i] and p[i][j] == col[j]:
                s += 1
    q.write(str(s))

File: 0301_0400/0388/0388.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m, **p, k = 0;

	fscanf(f, "%d%d", &n, &m);
	p = (int**)malloc(sizeof(int*) * (n + 1));
	for (int i = 0; i < n; ++i)
	{
		p[i] = (int*)malloc(sizeof(int) * (m + 1));
		fscanf(f, "%d", p[i]);
		p[i][m] = p[i][0];
		for (int j = 1; j < m; ++j)
		{
			fscanf(f, "%d", p[i] + j);
			if (p[i][j] < p[i][m])
				p[i][m] = p[i][j];
		}
	}
	p[n] = (int*)malloc(sizeof(int) * m);
	for (int j = 0; j < m; ++j)
	{
		p[n][j] = p[0][m];
		for (int i = 0; i < n; ++i)
			if (p[i][j] > p[n][j])
				p[n][j] = p[i][j];
	}
	for (int i = 0; i < n; ++i)
		for (int j = 0; j < m; ++j)
			if (p[i][j] == p[n][j] && p[i][j] == p[i][m])
				++k;
	fprintf(q, "%d", k);
	return 0;
}
